README.md

```diff
@@ -1 +1,52 @@
-# Hello-Micronaunts
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/AbdimulhinYussuf3675)
+- Twitter: [@twitterhandle](https://twitter.com/abdimulhin)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdimulhin-yussuf-7b110720b) 
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/AbdimulhinYussuf3675/Hello-Micronaunts/issues/)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
The link to the license is not working. You can fix this by creating a 'LICENSE' file (in the same location as the 'Readme.md' file), copy [this](https://github.com/jerryowusu/Web-Summit-Accra/blob/main/LICENSE) template into it and edit the year and name accordingly.